R/addin.R

```diff
@@ -0,0 +1,92 @@
+package_search <- function() {
+  if (!requireNamespace("shiny", quietly = TRUE)) {
+    stop("The shiny package is required for this gadget to work. Please install it.",
+         call. = FALSE
+    )
+  }
+  
+  if (!requireNamespace("miniUI", quietly = TRUE)) {
+    stop("The miniUI package is required for this gadget to work. Please install it.",
+         call. = FALSE
+    )
+  }  
+  
+  if (!requireNamespace("DT", quietly = TRUE)) {
+    stop("The DT package is required for this gadget to work. Please install it.",
+         call. = FALSE
+    )
+  }  
+  
+  if (!requireNamespace("rstudioapi", quietly = TRUE)) {
+    stop("The rstudioapi package is required for this gadget to work. Please install it.",
```

Comment:
I'm not sure it makes sense to write this here, but then, if rstudiapi is in Suggests only, I'm not sure where to check it's installed. :thinking: